cmd/commento/main.go

```diff
@@ -15,23 +16,23 @@ func main() {
 	if err != nil {
 		Die(err)
 	}
-	
+
+	// Load configuration from the environment.
```

Comment:
The env file can be specified explicitly e.g. via a env variable or a flag, and default to `.env` .
e.g.
A global `$COMMENTOENV=development.local` will provide specificity, where the env file will be `".env."+os.Getenv("COMMENTOENV")`, and preventing best-guess config selection using this loop.